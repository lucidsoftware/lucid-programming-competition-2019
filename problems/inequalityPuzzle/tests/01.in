|||||<||||||>|||||||<|>|||<||||||||<||<<
0000000030000000000000000
