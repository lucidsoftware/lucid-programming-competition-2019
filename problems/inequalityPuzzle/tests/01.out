1425321435523414351235124
